Makefile

```diff
@@ -69,6 +75,14 @@ docs: $(docs-targets)
 start-docker:
 	@docker compose up --remove-orphans
 
+start-docker-e2e:
+	@docker compose -f tests/e2e/docker-compose.e2e.yml up -d
+
+start-e2e: export HOST_UID=$(id -u)
+start-e2e: export HOST_GID=$(id -g)
+start-e2e: start-docker-e2e
+	docker exec oasis-indexer sh -c "cd /oasis-indexer && make test-e2e"
```

Comment:
Multiple targets for `start-e2e`; typo? The last rule will overwrite the others.

Also nit: I'm planning to update the dockerfile with `WORKDIR /oasis-indexer` so I'll hopefully be able to simplify the command

Reply:
Removed as unused.